templates/slots.html

```diff
@@ -42,6 +42,9 @@
                 console.error("error loading table state, err:", data, "status: ", status)
               }
               var response = data !== "" ? data : JSON.parse(localStorage.getItem("DataTables_" + settings.sInstance))
+              if (q) {
+                response.search.search = q
+              }
```

Comment:
This serves to ensure that use the query from the URL if there is one instead of info from the db or local storage.